server-test/src/test/scala/testpkg/BuildServerTest.scala

```diff
@@ -108,6 +108,18 @@ object BuildServerTest extends AbstractServerTest {
     })
   }
 
+  test("buildTarget/cleanCache") { _ =>
+    val buildTarget = buildTargetUri("util", "Compile")
+    svr.sendJsonRpc(
+      s"""{ "jsonrpc": "2.0", "id": "32", "method": "buildTarget/cleanCache", "params": {
```

Comment:
Not a big deal but every request should have a unique id. Here `32` is already used by the `buildTarget/compile` request.

```suggestion
      s"""{ "jsonrpc": "2.0", "id": "44", "method": "buildTarget/cleanCache", "params": {
```